.travis.yml

```diff
@@ -43,6 +43,7 @@ matrix:
           env: TOXENV=py36
 
 install:
+    - git fetch --unshallow --tags
```

Comment:
correct, anther option would be to pass in an arbitrary depth to the travis config,
i simply choose this one as its more defensive and paranoid (unshallow + fetch tags)